_includes/install/sdk.md

```diff
@@ -1,8 +1,8 @@
 ### Step 1: Download the Self-hosted package
 
-If you'd rather download and install the script manually, get the package from [TinyMCE Downloads](https://www.tinymce.com/download).
+If you'd rather download and install the script manually, get the package from [{{site.productname}} Downloads]({{site.gettiny}}/).
```

Comment:
Should the forward slash be here? as that seems like it could produce a strange or invalid URL.

Reply:
fixed